Standards/CCSMB-5.md

```diff
@@ -0,0 +1,22 @@
+# CCSMB-5: Radio Transmissions
+```md
+*Author: Emma Beurskens <@emmaknijn>*
+*Version: v1.0.0*
+*Last updated: YYYY-MM-DD*
+```
+
+## Design
+A client will only listen to packets while the audio is being transmitted from a server
+The server will communicate to the client over modem
+
+## Packet
+A packet consists of the following fields in a table:
+- buffer: Signed 8-bit PCM @ 48kHz
+- id: A random number that identifies the radio station
```

Comment:
A random number? Is the `stationName` not usable as identification?

Reply:
stationName could be used to also show stuff like the current show, as where id should be persistent over time

Reply:
Yeah. In the United States, radio stations broadcast their call sign and sometimes the title and author of the playing song.

Reply:
Considering we have title now, I think id could be dropped, since title would be usable for that.